src/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.cpp

```diff
@@ -33,7 +33,8 @@ using ostk::astro::trajectory::state::CoordinatesSubset;
 const Shared<const Frame> Propagator::IntegrationFrameSPtr = Frame::GCRF();
 
 Propagator::Propagator(const NumericalSolver& aNumericalSolver, const Array<Shared<Dynamics>>& aDynamicsArray)
-    : dynamicsContexts_(),
+    : stateBuilder_(Propagator::IntegrationFrameSPtr, coordinatesBrokerSPtr_),
```

Comment:
Worth noting that the coordinatesBroker_ here will be nullptr.
It gets populated downstream but just thought it was worth mentioning.
I think it's fine cause it's a shared pointer.

Reply:
I believe when we call `std::make_shared` ([here](https://github.com/open-space-collective/open-space-toolkit-astrodynamics/blob/main/include/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.hpp#L230)) with no arguments, it calls the default constructor. 


Check this out:
https://cpp.sh/
```
#include <iostream>
#include <memory>
int main () {
  struct thing {
   int i = 1;
   int j = 2;
  };
  std::shared_ptr<thing> foo = std::make_shared<thing> ();
std::cout << "*foo: " << foo->i << '\n';
std::cout << "*foo: " << foo->j << '\n';
  return 0;
}
```

Reply:
Yeah good point it'll just be empty. So when we add subsets it'll be shared.